### PR TITLE
Feature | IATI Gates

### PR DESCRIPTION
**Pull request by dean-breed**
@edwinmp these are the two scripts I'm looking to add. I am trying to use git2r package which seems to work in general but I will need to supply credentials in some form.

### CODE REVIEW

**Comment by dean-breed** [comment]
@edwinmp One for Monday but this is the Pr with the new Python scripts

**Comment by dean-breed** [comment]
@edwinmp @wakibi Thanks for all your help with this!

**Comment by edwinmp** [comment]
@dean-breed are you still actively working on this or is it ready for review?

**Comment by dean-breed** [comment]
@edwinmp Still working on it but should be able to get it done today!

**Comment by dean-breed** [comment]
@edwinmp done from my end

File: data_updates/Python/csv/one.csv [hunk 1]
```diff
@@ -0,0 +1,5 @@
+etea
+dgesdg
+agesdg
+segrsagre
+aegrgr
```
Comment (edwinmp) [inline data_updates/Python/csv/one.csv]: What's the purpose of this file? It's not proper CSV & isn't being used or generated anywhere.

File: data_updates/Python/iati_rhfp.py [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+
+# print(data)
+
+# # Put it in millions
+
+# data["x_transaction_value_usd_m_Sum"] = data["x_transaction_value_usd_m_Sum"]/1000000
+
+# data = data[data['country'] != "Germany"]
+
+# data.to_csv("csv/iati_rhfp3.csv", encoding='utf-8', index=False)
```
Comment (edwinmp) [inline data_updates/Python/iati_rhfp.py]: Any chance we can get rid of this large section of commented out code, or will it be useful at some point?

**Review by edwinmp — CHANGES_REQUESTED** [review]
We're not using the R files anymore, so please delete them.
Also, the generated CSV files are only useful for deploying to the `di-website-data` repo so there's no value in committing them. Delete them, commit, then add the folder to `.gitignore` ... I suggest u name the csv folder something more specific like `iati_csv`, otherwise someone may use that same name for something else & we'll end up with their files in the data repo :)

**Comment by wakibi** [comment]
@dean-breed you have feedback you have not yet worked on. Any chance this can be wrapped up this week?

**Comment by dean-breed** [comment]
@wakibi There are lots of deadlines this week for me but I will be able to do it by the end of the week. Chase me up if I haven't done it by Friday morning! Cheers

File: data_updates/Python/csv/one.csv (hunk 1 above)
Comment (dean-breed) [reply data_updates/Python/csv/one.csv]: I think this was sometime @wakibi did for testing but can delete?

File: data_updates/Python/iati_rhfp.py (hunk 2 above)
Comment (dean-breed) [reply data_updates/Python/iati_rhfp.py]: It was for a CSV we stopped using so yes. Deleted.

**Comment by dean-breed** [comment]
@edwinmp @wakibi Sorry for the delay in making these changes but think they have all been processed now. Cheers for your time reviewing them!

File: data_updates/Python/iati_rhfp.py [hunk 3]
```diff
@@ -0,0 +1,168 @@
+import os
+import requests
+import glob
+from requests.auth import HTTPBasicAuth
+import pandas as pd
+from github import Github, InputGitTreeElement
+from django.conf import settings
+
+
+donors_selected = pd.read_csv("https://staging-ddw.devinit.org/api/export/1232")
```
Comment (edwinmp) [inline data_updates/Python/iati_rhfp.py]: Is this meant to be a staging URL or was that just for testing?

Comment (dean-breed) [reply data_updates/Python/iati_rhfp.py]: Resolved below @edwinmp

**Review by edwinmp — APPROVED** [review]

**Pull request merged**